pkg/runcfanotify/runcfanotify.go

```diff
@@ -100,6 +100,14 @@ func Supported() bool {
 	if !runcFound {
 		return false
 	}
+	// Test that it's possible to run fanotify
+	fanotifyFlags := uint(unix.FAN_CLOEXEC | unix.FAN_CLASS_CONTENT | unix.FAN_UNLIMITED_QUEUE | unix.FAN_UNLIMITED_MARKS)
+	openFlags := os.O_RDONLY | unix.O_LARGEFILE | unix.O_CLOEXEC
+	notifyFD, err := fanotify.Initialize(fanotifyFlags, openFlags)
```

Comment:
What about moving this into an `initFanotify()` function that would be also called by `NewRuncNotifier()`? By doing so, we will be sure we are trying to initialize it with the same flags/options we will initialize it later. WDYT?

Reply:
Done, thanks.